src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -375,14 +376,16 @@ def build_actions(
     group: Group,
     project: Project,
     text: str,
-    color: str,
     actions: Sequence[MessageAction] | None = None,
     identity: RpcIdentity | None = None,
-) -> tuple[Sequence[MessageAction], str, str]:
+) -> tuple[Sequence[MessageAction], str, bool]:
     """Having actions means a button will be shown on the Slack message e.g. ignore, resolve, assign."""
     if actions and identity:
         text = get_action_text(text, actions, identity)
-        return [], text, "_actioned_issue"
```

Comment:
this was previously used to change the bar color to white. but we aren't hitting this anymore because we pass `issue_details=False` for any action taken from slack...

Reply:
the default for the `issue_details` argument is `False`